### ISSUE TITLE
Allow configuring the format for date/datetime fields.

### ISSUE THREAD

**Issue by ADmad** [sulu/sulu#4245]
| Q | A
| --- | ---
| Bug? | no
| New Feature? | yes
| Sulu Version | 2.0.0-alpha4
| Browser Version | N/A

#### Actual Behavior

Currently for date/datetime fields the display string is formatted based on browser's locale, e.g. "Authored date" fields on "Settings" tab for Pages and there's no way to customize it.

#### Expected Behavior

To be able to configure the date/datetime format instead of relying on browser locale.

**Comment by danrot** [comment]
That's something we have already talked about. The problem is that we need another setting in Sulu for that, because we need to include the country along with the language (different english speaking countries still have different date formats etc.). I think that feature won't make it into 2.0, because of our limited time, but we might maybe look at it in Sulu 2.1.

**Comment by ADmad** [comment]
Yes you would have to include country and language. Using `intl` extension and the `Locale` class it provides would help with that which allows you use locale identifier like `en_UK` for e.g. to indicating language and region.
